compiler/src/iree/compiler/Codegen/Common/TransformExtensions/CommonExtensionsOps.td

```diff
@@ -153,4 +153,50 @@ def ForeachThreadToWorkgroupOp : Op<Transform_Dialect,
   }];
 }
 
+def TileToWorkgroupsOp :
+    Op<Transform_Dialect, "iree.tile_to_workgroups_op",
+      [AttrSizedOperandSegments,
+       DeclareOpInterfaceMethods<MemoryEffectsOpInterface>,
+       TransformOpInterface]> {
+  let description = [{
+    Wrapper around `structured.tile_to_foreach_thread_op` for use within IREE.
+
+    In addition to tile and distribute using `scf.foreach_thread`, lowers the
```

Comment:
Note that this is not the same "tile and distribute" you have been accustomed to, the thread ids here are virtual and are represented by bbArgs.
There is a further separate step of lowering to concrete workgroup/thread ids.